src/tests/JIT/Regression/JitBlue/GitHub_1161/GitHub_1161.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 // This test illustrates a limitation in the JIT in that it will not promote
-// a struct that has a single double register. See https://github.com/dotnet/coreclr/issues/1161.
+// a struct that has a single double register. See https://github.com/dotnet/runtime/issues/4323.
```

Comment:
It looks weird that the test is called GItHut_1161, but points to issue 4323 now. What's wrong with the original link? The archived CoreCLR repo is not going to disappear suddenly.